e2e_testing/torchscript/slice_like.py

```diff
@@ -0,0 +1,203 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+import torch
+
+from torch_mlir_e2e_test.torchscript.framework import TestUtils
+from torch_mlir_e2e_test.torchscript.registry import register_test_case
+from torch_mlir_e2e_test.torchscript.annotations import annotate_args, export
+
+# ==============================================================================
+
+class SliceModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return x[0:5:1, 1:3:1, 2:4:1]
+
+
+@register_test_case(module_factory=lambda: SliceModule())
+def SliceModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(6,4,7))
+
+
+
+# ==============================================================================
+
+class SliceModuleLowStartIndex(torch.nn.Module):
```

Comment:
nit: `LowStart` => `NegStart`. Also, for consistency, we usually put `Module` at the end. `SliceNegStartModule`

Reply:
Changed the name if you can think of a shorter one, lmk. Its designed to test the case where start+dimSize < 0